source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.yml

```diff
@@ -0,0 +1,354 @@
+                  retryWrites: true
+                  minPoolSize: 0
+            - database:
+                id: &database database
+                client: *client
+                databaseName: *databaseName
+            - collection:
+                id: &collection collection
+                database: *database
+                collectionName: *collectionName
+            - thread:
+                id: &thread1 thread1
+      # ensure the primary is discovered                
+      - name: insertOne
+        object: *collection
+        arguments:
+          document: { _id: 1 }
+      # simulate a long-running query
+      - name: runOnThread
+        object: testRunner
+        arguments:
+          thread: *thread1
+          operation:
+            name: updateOne
+            object: *collection
+            arguments:
+              filter: 
+                $where: sleep(2000) || true
+              update: 
+                $set: { "a": "bar" }
+      - name: waitForEvent
+        object: testRunner
+        arguments:
+          client: *client
+          event:
+            commandStartedEvent: {}
+          count: 2
+        # Configure the monitor check to fail with a timeout.
+        # Use times: 2 to account for the RTT monitor triggering the failpoint instead of the regular monitor
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: *setupClient
+          failPoint:
+            configureFailPoint: failCommand
+            mode:
+              times: 2
+            data:
+              failCommands:
+                - hello
+                - isMaster
+              blockConnection:  true,
+              blockTimeMS: 1500,                
+              appName: interruptInUseRetryableWrite
+      - name: waitForThread
+        object: testRunner,
+        arguments:
+          thread: *thread1
+
+    expectEvents:
+      - client: *client
+        eventType: command
+        events:
+          - commandStartedEvent:
+              commandName: insert
+          - commandSucceededEvent:
+              commandName: insert
+          - commandStartedEvent:
+              commandName: update
+          - commandFailedEvent:
+              commandName: update
+          - commandStartedEvent:
+              commandName: update
+          - commandSucceededEvent:
+              commandName: update
+      - client: *client
+        eventType: cmap
+        events:
+          - connectionCheckedOutEvent: {}
+          - connectionCheckedInEvent: {}
+          - connectionCheckedOutEvent: {}
+          - poolClearedEvent:
+              interruptInUseConnections: true
+          - connectionCheckedInEvent: {}
+          - connectionClosedEvent: {}
+          - connectionCheckedOutEvent: {}
+          - connectionCheckedInEvent: {}
+     
+    outcome:
+      - collectionName: *collectionName
+        databaseName: *databaseName
+        documents:
+          - _id: 1, a : bar
```

Comment:
These need to be surrounded in braces or on separate lines. See the GitHub action failure for details.

Reply:
done